guide/storing_data.rst

```diff
@@ -60,3 +61,24 @@ Example:
   foo.bar = 1
   foo = {"bar":1,"baz":{"newBaz":true}}
   */
+
+Addon data: working with repository of models
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``addonData`` allows to add or retrieve (all or filtered by criteria) models, stored on server:
+
+* ``define(modelName, schema)`` - defines model ``modelName`` with schema ``schema``. Schema implements types ``String``, ``Number``, ``Object``, ``Boolean``
+* ``getRepository(modelName)`` - return repository of models stored as ``modelName`` that can be user to retrieve, save or delete ``modelName`` models
```

Comment:
... returns repository ...
... can be used ...

I think it is better to use 'defined model' instead of 'models stored as'